README.md

```diff
@@ -63,7 +63,7 @@ To configure the Welcome Bot, edit your `config.json` file with a message you wa
 
 where
 
-- **TeamName**: The team for which the Welcome Bot sends a message for. Must be the team handle used in the URL, in lowercase. For example, in the following URL the **TeamName** value is `my-team`: https://example.com/my-team/channels/my-channel
+- **TeamName**: The teams for which the Welcome Bot sends a message for. Must be the team handle used in the URL, in lowercase. For example, in the following URL the **TeamName** value is `my-team`: https://example.com/my-team/channels/my-channel . In case of multiple teams, ensure that the comma seperated fields dont have any white space between them. For example `"my-team,my-team-2"` is valid while `"my-team, my-team-2"` is not valid. 
```

Comment:
Can we be more forgiving and allow a (or multiple) whit spaces before and after the comma.

Reply:
@hanzei  I have pushed a fixe to allow whitespaces.